requirements.txt

```diff
@@ -10,6 +10,7 @@ packaging
 pamela>=1.1.0; sys_platform != 'win32'
 prometheus_client>=0.5.0
 psutil>=5.6.5; sys_platform == 'win32'
+pydantic>=2
```

Comment:
Since it comes into a new major release, I would say it is fine to keep things simple and update pydantic.

Reply:
Let's go with >2 straight away as its a major release. I see that it has been around since Jun 30, 2023, so its not super new. If there is another software installed next to jupyterhub, it may very well have a release out that supports pydantic 2 at this point.